extension/lib/magento/customer/deleteItems.js

```diff
@@ -0,0 +1,32 @@
+const MagentoRequest = require('../Request')
+
+/**
+ * @param {StepContext} context
+ * @param {Object} input
+ * @param {string[]} input.productIds - array of ids of the products to remove
+ * @param {string} input.token - user token for authentication
+ */
+module.exports = async (context, input) => {
+  const wishlistItemIdMapping = await context.storage.user.get('wishlistItemIdMapping')
+  const wishlistItemIdsString = input.productIds.map(productId => wishlistItemIdMapping[productId]).toString()
+
+  if (!wishlistItemIdsString) {
+    return
+  }
+
+  const request = new MagentoRequest(context, input.token)
+  /**
+   * Get all wishlist ids of the customer. At the moment we only support one wishlist per customer
+   * @typedef {Object} wishlists
+   * @property {string} wishlist_id
+   */
+  const wishlists = await request.send(`${context.config.magentoUrl}/wishlists`, 'getWishlistItems')
```

Comment:
as mentioned before, this will need to change in accordance with Alexander's implementation. This also includes creating a different pipeline and using it in the then/else statement